docs/stopping_events/breakpoints.md

```diff
@@ -0,0 +1,109 @@
+---
+icon: material/sign-caution
+search:
+    boost: 4
+---
+# :material-sign-caution: Breakpoints
+Breakpoints are the killer feature of any debugger, the fundamental stopping event. They allow you to stop the execution of your code at a specific point and inspect the state of your program to find bugs or understand its design.
+
+!!! WARNING "Multithreading and Breakpoints"
+    **libdebug** breakpoints are shared across all threads. This means that if a breakpoint is hit in one thread, all threads will stop. You can use the [`hit_on()`](../debugging_flow/#hit-records) method of a breakpoint object to determine if a breakpoint was hit in a specific thread.
```

Comment:
IMO they are two different concepts

Reply:
Absolutely fair, on second read it is was mixing things up

Reply:
Addressed in [9e4a7f8](https://github.com/libdebug/libdebug/pull/129/commits/9e4a7f831a709f7ca168593df7deb81678b86ed5)